filter/expand_json.go

```diff
@@ -0,0 +1,133 @@
+package filter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/AdRoll/baker"
+	"github.com/jmespath/go-jmespath"
+)
+
+var ExpandJSONDesc = baker.FilterDesc{
+	Name:   "ExpandJSON",
+	New:    NewExpandJSON,
+	Config: &ExpandJSONConfig{},
+	Help:   "Explodes json objects to other fields.",
```

Comment:
- We can expand a bit the help string, maybe also including a link to https://jmespath.org/.
you can use multi-lines here if needed.

- Not sure about using explodes here